liquidcore/config/serializers.py

```diff
@@ -0,0 +1,124 @@
+import re
+from django.contrib.auth.models import User
+from rest_framework import serializers
+
+from .models import *
+
+def valid_username(username):
+    if not username:
+        raise serializers.ValidationError("empty username")
+    if not bool(re.match(r"^(\w|\.)+$", username)):
+        raise serializers.ValidationError("username can only contain letters, digits, underscore and dot")
+    if len(username) > 64:
```

Comment:
Do we have a minimum username length restriction? I think Hypothesis requires at least 2 letters.

Reply:
I say we fix it at 3 characters just to be safe.

Reply:
Sold, to the man with three fingers!